stream/feed.py

```diff
@@ -46,6 +46,9 @@ def add_activity(self, activity_data):
             activity_data = {'actor': 1, 'verb': 'tweet', 'object': 1}
             activity_id = feed.add_activity(activity_data)
         '''
+        if activity_data.get('to') and type(activity_data.get('to')) is str:
```

Comment:
why not checking for the excepted type directly?

```pyhton
isinstance(activity_data.get('to'), (list, tuple, set))
```

for the record, here's a safer way to check for strings:
```python
import six

if isinstance(activity_data.get('to'), six.string_types):
```